### PR TITLE
Redis 6.0.1 in builders

### PR DESCRIPTION
**Pull request by rafie**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,21 +9,22 @@ ARG OSNICK=buster
 ARG ARCH=x64
 
 #----------------------------------------------------------------------------------------------
-FROM redisfab/redis:6.0-rc3-${ARCH}-${OSNICK} AS builder
+FROM redisfab/redis:${REDIS_VER}-${ARCH}-${OSNICK} AS builder
+
+ARG REDIS_VER
```
Comment (ashtul) [inline Dockerfile:14]: Why is this here? and in line 27?

Comment (rafie) [reply Dockerfile:14]: It's the way Docker handles arguments. You need to re-declare them after each FROM clause. Really annoying.
